domain/notification/handler.go

```diff
@@ -42,7 +42,7 @@ func NewHandler(auth auth.Service, localFHIRClientFactory fhir.Factory, tranferR
 func (service *handler) Handle(ctx context.Context, notification Notification) error {
 	fhirServer, err := service.registry.GetCompoundServiceEndpoint(ctx, notification.SenderDID, "eOverdracht-sender", "fhir")
 	if err != nil {
-		return fmt.Errorf("error while looking up custodian's FHIR server (did=%s): %w", notification.SenderDID, err)
+		return fmt.Errorf("error while looking up authorizer's FHIR server (did=%s): %w", notification.SenderDID, err)
```

Comment:
this would actually be custodian, since in the context of data it's no longer an authorizer but data provider